DocumentsFromSnapshotMigration/src/main/java/org/opensearch/migrations/RfsMigrateDocuments.java

```diff
@@ -76,93 +76,93 @@ public static class Args {
         private boolean help;
 
         @Parameter(required = true,
-            names = { "--snapshot-name" },
+            names = { "--snapshot-name", "--snapshotName" },
```

Comment:
This pattern of changes seems like it exposes a large risk for some commands to be missed which will create headaches when a parameter value isn't found.

We could ensure this behavior would be by wrapping the behavior at a higher level than on each individual parameter, or by having a way (test?) we are certain every parameter has a kabob-case and camelCase version.

Reply:
I'd love to investigate (& hopefully DO!) doing this at a higher level.  I honestly needed to try something out early on and each time that I had to do this, it was just ~5 minutes more - but in aggregate, it was considerable & the maintenance story sucks.
Here's a [jira](https://opensearch.atlassian.net/browse/MIGRATIONS-2314) to make this easier.